addons/sourcemod/scripting/Extend.sp

```diff
@@ -67,21 +67,22 @@ public void OnPluginStart()
 	{
 		RegAdminCmd("sm_extendmap", Command_Extend, ADMFLAG_GENERIC, "Add more time to mp_timelimit");
 		RegAdminCmd("sm_extend", Command_ExtendVote, ADMFLAG_GENERIC, "sm_extend [time] - Start an extendvote");
+		RegAdminCmd("sm_roundextend", Command_RoundExtend, ADMFLAG_GENERIC, "sm_roundextend - Extend mp_timelimit elapsed time since the start of the round");
```

Comment:
whats the purpose of this command

Reply:
In case of round ruined by strollers for example, admin can extend the map for the time elapsed from the round since it's started.

We can allow this cmd for donated admins like that and revoke them extendmap (where they can do what they want of the map time)

Reply:
ohh i see nice!